File: lab2.py
while True:
 x = int(input("Give a number greater than 9:"))
 if x > 10:
    break


count = 0
while x >= 10:

 old = x
 sum = 0

 while old > 0:
     rakam = old % 10
     sum = sum + rakam
     old = old // 10

 print(x, "→", sum)
 count+=1
 x = sum

print(f'Last value: {x}')
print(f'Total steps: {count}')

File: lab2.cpp
#include <iostream>
using namespace std;

int main() {
    int count = 0;
int n;
    cout << "Please enter a positive integer greater than 9:";
    cin >> n;

    while (n >= 10) {
     int temp = n;
        int sum = 0;
        while (temp > 0) {
            sum += temp % 10;
            temp = temp / 10;
        }
        cout << n << "=>" << sum << endl;
        n = sum;
        count++;
    }
    cout << "Last value: " << n << endl;
    cout << "Total steps: " << count << endl;

return 0;
}